src/lib/Tab/LocationView/DetailsTab.php

```diff
@@ -139,10 +138,22 @@ public function getTemplateParameters(array $contextParameters = []): array
         $lastContributor = (new UserExists($this->userService))->isSatisfiedBy($versionInfo->creatorId)
             ? $this->userService->loadUser($versionInfo->creatorId) : null;
 
-        $canSeeSection = $this->permissionResolver->hasAccess('section', 'view');
+        $section = null;
+        $canSeeSection = $this->permissionResolver->hasAccess('section', 'view') !== false;
```

Comment:
array is returned only if limitations are on this function, so we can not expect empty array here